airflow/api_connexion/endpoints/event_log_endpoint.py

```diff
@@ -69,9 +69,11 @@ def get_event_logs(
         "execution_date",
         "owner",
         "extra",
+        "run_id",
     ]
     total_entries = session.scalars(func.count(Log.id)).one()
-    query = select(Log)
+    run_id_alias = aliased(Log)
+    query = select(Log, run_id_alias.run_id.label("run_id"))
```

Comment:
I'm confused, how it expected to be work? `Log` doesn't have `run_id`

https://github.com/apache/airflow/blob/ae3a61775a79a3000df0a8bdf50807033f4e3cdc/airflow/models/log.py#L27-L40

Reply:
Oops i forgot to update that !!

Reply:
On it

Reply:
hey @Taragolis  I have just updated it but  i was unable to commit it this is what i got I tried to rebase and remove any coflicts if the branch had still this error persuade
Need Help here 
 ! [rejected]              NewBranch34211 -> NewBranch34211 (non-fast-forward)
error: failed to push some refs to 'https://github.com/Vidhanvyrs/airflow.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Reply:
I thought about avoiding adding a new column to Log and doing a join with DagRun to retrieve the run_id, but since we are discussing removing the run_date unique constraint, I think adding a new column is the best choice here.